README.md

```diff
@@ -72,75 +72,108 @@ For guidance on developing for PowerToys, please read the [developer docs](/doc/
+- Fix issues causing windows not to be restored correctly to their last known zone.
 
 ### File explorer add-ons
 
-- Fixed escaping HTML-sensitive characters when previewing developer files. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
+- Add Developer files previewer option to set max file size and fix styling issue. Thanks [@Aaron-Junker](https://github.com/Aaron-Junker)!
+- Improve Developer files previewer exception handling and printing of error messages.
+- Fix crash when generating PDF and Gcode files thumbnails. (This was a hotfix for 0.67)
 
-### Image Resizer
+### Hosts file editor
 
-- Improved code quality around a silent crash that was being reported to Microsoft servers.
+- Improve hosts file loading. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
+- Improved duplicate hosts finding. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
+
+### Keyboard Manager
+
+- Fix typo in Keyboard Manager Editor. Thanks [@ChristianLW](https://github.com/ChristianLW)!
+
+### Mouse Utils
+
+- Resolve grammatical error in Mouse Highlighter description. Thanks [@WordlessSafe1](https://github.com/WordlessSafe1)!
+- New utility: Mouse Jump allows to quickly move the mouse pointer long distances on single or across screens. Thanks [@mikeclayton](https://github.com/mikeclayton)!
+
+### Paste as Plain Text
+
+- New utility: Paste as Plain Text allows pasting the text contents of your clipboard without formatting. Note: the formatted text in the clipboard is replaced with the unformatted text. Thanks [@carlos-zamora](https://github.com/carlos-zamora)!
 
 ### PowerToys Run
 
-- Add option to tab through results only. Thanks [@maws6502](https://github.com/maws6502)!
-- System plugin - Updated Recycle Bin command to allow opening the Recycle Bin. Thanks [@htcfreek](https://github.com/htcfreek)!
-- System plugin - Improved Recycle Bin command to not block PT Run while the deletion is running. Thanks [@htcfreek](https://github.com/htcfreek)!
-- System plugin - Small other changes to improve the usability of the Recycle Bin command. Thanks [@htcfreek](https://github.com/htcfreek)!
-- WindowWalker plugin - Show all open windows with action keyword. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
+- Show Steam (steam://open/) shortcuts in the Program plugin. 
+- Localize paths of Program plugin results. Thanks [@htcfreek](https://github.com/htcfreek)!
+- Support MSC and CPL files in "Run command" results of Program plugin. Thanks [@htcfreek](https://github.com/htcfreek)!
+- Added missing MSC and CPL settings to the results of Windows Settings plugin. Thanks [@htcfreek](https://github.com/htcfreek)!
+- System plugin: Setting for separate "Open/Empty Recycle bin" results or single result with context menu. Thanks [@htcfreek](https://github.com/htcfreek)!
+- Add support for log2 and log10 in Calculator plugin. Thanks [@RickLuiken](https://github.com/RickLuiken)!
+- Removed the TimeZone plugin.
```

Comment:
```suggestion
- Removed the TimeZone plugin. (We hope to bring it back again after fixing the bugs.)
```

Reply:
it is more than a few bugs to bring it back.